src/Mvc/Mvc.Core/src/ConsumesAttribute.cs

```diff
@@ -53,6 +55,31 @@ public ConsumesAttribute(string contentType, params string[] otherContentTypes)
             ContentTypes = GetContentTypes(contentType, otherContentTypes);
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="ConsumesAttribute"/>.
+        /// </summary>
```

Comment:
Could we document the parameters? I know the other constructor is in poor showing by not doing so already.

Reply:
yes, just did.